lib/brains/errors.py

```diff
@@ -6,6 +6,13 @@ class MockError(Exception):
     """
 
 
+class BraintreeError(Exception):
+
+    """
+    A standard Braintree error.
+    """
+
+
 class BraintreeResultError(Exception):
```

Comment:
BraintreeError should be removed, it was something I was playing with before I simplified.